src/Definition.elm

```diff
@@ -127,22 +127,57 @@ namespaceAndOtherNames suffixName fqns =
 -- VIEW
 
 
+viewBuiltinBadge : String -> Category -> Html msg
+viewBuiltinBadge name_ category =
+    let
+        content =
+            span
+                []
+                [ strong [] [ text name_ ]
+                , text " is a "
+                , strong [] [ text ("built-in " ++ Category.name category) ]
+                , text " provided by the Unison runtime"
+                ]
+    in
+    UI.badge content
+
+
+viewBuiltin : Definition -> Html msg
+viewBuiltin definition =
+    case definition of
+        Term _ info ->
+            case info.source of
+                BuiltinTerm _ ->
+                    div [ class "built-in" ] [ viewBuiltinBadge info.name (Category.Term Category.PlainTerm) ]
```

Comment:
This is hardcoded right now, should eventually come from the definition itself when that is in the payload